src/controller/python/OpCredsBinding.cpp

```diff
@@ -149,6 +151,33 @@ class TestCommissioner : public chip::Controller::AutoCommissioner
             return CHIP_NO_ERROR;
         }
 
+        if (report.stageCompleted == chip::Controller::CommissioningStage::kGenerateNOCChain)
+        {
+            if (report.Is<chip::Controller::NocChain>())
+            {
+
+                auto nocChain = report.Get<chip::Controller::NocChain>().rcac;
+                MutableByteSpan rcacSpan(const_cast<uint8_t *>(nocChain.data()), nocChain.size());
```

Comment:
Does this get used anywhere besides creating the non-mutable span below? Is it possible to get rid of this? Does it actually need to be mutable? i don't think the conversion is in-place.

Reply:
Thank you, you are right, I did not need to create the mutable span in the code to get this to work. 
I have updated the code to just converting the x509 cert data stored in nocChain to the chipRcacSpan mutable span now.